src/app/_components/DesktopNavigation.tsx

```diff
@@ -95,11 +97,11 @@ function ExternalLink({ label, description, href }: LinkProps) {
 }
 
 const getInvolvedInternalItems = getInvolvedItems.filter((item) =>
-  item.href.startsWith('/'),
+  isInternalLink(item.href),
 )
 
-const getInvolvedExternalItems = getInvolvedItems.filter(
-  (item) => item.href.startsWith('https://') || item.href.startsWith('http://'),
+const getInvolvedExternalItems = getInvolvedItems.filter((item) =>
+  isExternalLink(item.href),
 )
```

Comment:
Wouldn't this be `!isInternLink` ?

Reply:
Would you prefer `!isInternLink` over `isExternalLink`?

Reply:
The current implementation of  `isExternalLink` seems a bit basic and limited? What if a URL starts with `www`?

```
export function isExternalLink(href: Link) {
  return href.startsWith('http://') || href.startsWith('https://')
}
```

If `isInternalLink` already exists - but you still wanted to be more explicit - You could always do:

```
export function isExternalLink(href: string) {
  return !isInternalLink(href)
}
```

Perhaps it helps with readability - I'm fine with both.